railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -34,7 +34,7 @@ group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   <%- if depends_on_system_test? -%>
   # Adds support for Capybara system testing and selenium driver
-  gem 'capybara', '~> 2.13.0'
+  gem 'capybara', '~> 2.13'
```

Comment:
Why had we chosen to pin new apps to 2.13.x?

Are we providing this as a conservative starting point, expecting Rails users to each figure out when+why to upgrade Capybara?